packages/drivers/local-driver/src/localDocumentService.ts

```diff
@@ -28,9 +28,14 @@ export class LocalDocumentService implements api.IDocumentService {
         private readonly tenantId: string,
         private readonly documentId: string,
         private readonly documentDeltaConnectionsMap: Map<string, LocalDocumentDeltaConnection>,
+        private readonly noDeltaStream: boolean,
```

Comment:
>noDeltaStream [](start = 25, length = 13)

since we use this a lot in tests, can we make this an optional override for all policies? so `private readonly policies: DocumentServicePolicies`